skrf/vi/scpi/parser.py

```diff
@@ -298,7 +298,7 @@ def parse_yaml_file(driver_yaml_file):
 
     driver_template = None
     with open(driver_yaml_file, 'r', encoding='utf-8') as yaml_file:
-        driver_template = yaml.load(yaml_file)
+        driver_template = yaml.load(yaml_file, Loader=yaml.Loader)
```

Comment:
would it give the same result using :
```suggestion
        driver_template = yaml.safe_load(yaml_file)
```
?
(https://pyyaml.docsforge.com/master/api/yaml/safe_load/)
If yes, this is maybe more explicit.

Reply:
That works but does create some potential issues with truncating zeros and possibly other background safeguards.

Question about this about the functionality you want users to have:
Should it be able to run python code when loading the .yaml file?
If not, then we might want to use the following:
`driver_template = yaml.load(yaml_file, Loader=yaml.BaseLoader)`
Otherwise, my original change using `yaml.Loader` will work best.

By default, the `yaml.load` methods Loader was `yaml.Loader` before this deprecation happened. `yaml.Loader` facilitates the execution of arbitrary Python code, `yaml.BaseLoader` doesn't allow for the execution of Python code. `yaml.SafeLoader` (which is used in `yaml.safe_load()`, doesn't allow code execution and has some other safeties I haven't looked into yet. The SafeLoader will also restrict your yaml tags to a degree which might limit your use cases.

Reply:
> That works but does create some potential issues with truncating zeros and possibly other background safeguards.

OK, fine. Let's keep it like that then.